### DISCUSSION TITLE
docs/3.x/api-usage/usage-limits

### DISCUSSION THREAD

**Discussion by giscus** [saleor/saleor-docs#1072] [General]
# docs/3.x/api-usage/usage-limits

Rate limits

https://docs.saleor.io/docs/3.x/api-usage/usage-limits

**Comment by lkostrowski** [comment]
This doc is specific to Saleor Cloud. Rate limiting in context of the raw Saleor is likely:
- what is the default
- how to change it

Information about sanboxes & prod, specific to Saleor Cloud, should be part of Saleor Cloud docs
